hack/collector-default

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
+# -------------------------------------------------------------------------------- #
+# collctor-default will collect host information as package for support-bundle-kit #
+# e.g., hostname, syslog, dmesg etc                                                #
+# -------------------------------------------------------------------------------- #
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+# -------------------------------------------------------------------------------- #
+# host information                                                                 #
+# -------------------------------------------------------------------------------- #
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+# CPU, Memory and Process usage status
+top -b -n 1 > cpu_memory_process_status
+
+# -------------------------------------------------------------------------------- #
+# collect logs                                                                     #
+# -------------------------------------------------------------------------------- #
+cd ${BUNDLE_DIR}
+mkdir -p logs
+cd logs
+
+dmesg &> dmesg.log
```

Comment:
Use `dmesg -T` for human-readable timestamps.

Reply:
Sorry, I missed the parameter and it has been fixed.

Reply:
I don't have permission to close the conversation. Free to close it, thank you.